cmd/search_log.go

```diff
@@ -54,6 +56,12 @@ var latencyAggregateCmd = &cobra.Command{
 	Run:   latencyAggregateFn,
 }
 
+var queriesAggregateCmd = &cobra.Command{
+	Use:   "queries_aggregate",
+	Short: "Get most popular queries from ElasticSearch.",
```

Comment:
Why popular? and not random?

Reply:
Text updated: Get 1000 most popular queries and 1000 random queries for each language from ElasticSearch.